platooning_control/include/platoon_control/platoon_control_worker.hpp

```diff
@@ -87,25 +82,13 @@ namespace platoon_control
         * \brief Generates speed commands (in m/s) based on the trajectory point
```

Comment:
Line 73 function's implementation is removed. Either add the implementation or remove this declaration
updateConfigParams